src/app/activity/bounty/Paid.ts

```diff
@@ -104,61 +107,88 @@ const writeDbHandler = async (request: PaidRequest, paidByUser: GuildMember): Pr
 
 	const dbBountyResult: BountyCollection = await bountyCollection.findOne({
 		_id: new mongo.ObjectId(request.bountyId),
-		status: BountyStatus.open,
+		status: { $nin: [BountyStatus.draft, BountyStatus.deleted] }
 	});
 
 	const currentDate = (new Date()).toISOString();
-	const writeResult: UpdateWriteOpResult = await bountyCollection.updateOne(dbBountyResult, {
+	// TODO: what's a better type here?
+	let writeObject: any = {
 		$set: {
-			reviewedBy: {
```

Comment:
Do we nee a conversion, or are IOUs not in production yet?

Reply:
IOUs are not in production - but re-included that field in the next revision.